webapp/src/dogma/common/components/editor/JsonPath.tsx

```diff
@@ -0,0 +1,11 @@
+import { DebouncedInput } from 'dogma/common/components/table/DebouncedInput';
+import { Box, Text } from '@chakra-ui/react';
+
+export const JsonPath = ({ handleQuery }: { handleQuery: (value: string | number) => void }) => {
+  return (
+    <Box mb="4">
+      <Text mb="2">Query by JSON Path</Text>
+      <DebouncedInput type="text" value="" onChange={handleQuery} placeholder={`$..[0]`} />
```

Comment:
While I tried to type some JSON path expressions, I got many error messages whenever I was in the middle of incomplete JSON path syntax.
I didn't see it as a good UX and it was annoying to me. 😅
What do you think of borrowing some ideas from https://jsonpath.com/ ?
Showing JSON expression tip looks good to me.

Reply:
Updated. 😅